sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_http_logging_policy.py

```diff
@@ -31,18 +30,33 @@
 import ast
 
 from azure.core.pipeline.policies import HttpLoggingPolicy
-
+from azure.cosmos.http_constants import HttpHeaders
 
 
 class CosmosHttpLoggingPolicy(HttpLoggingPolicy):
 
-    def __init__(self, logger=None, **kwargs): # pylint: disable=unused-argument
+    REDACTED_PLACEHOLDER = "REDACTED"
+
+    def __init__(self, logger=None, **kwargs):  # pylint: disable=unused-argument
         self._enable_diagnostics_logging = kwargs.pop("enable_diagnostics_logging", None)
         super().__init__(logger, **kwargs)
         if self._enable_diagnostics_logging:
             self.logger = logger
 
-    def on_request(self, request): # pylint: disable=too-many-return-statements, too-many-statements
+        self.default_headers_whitelist = list()
+        for k, v in HttpHeaders.__dict__.items():
+            if not k.startswith("_") and k != "Authorization":
+                self.default_headers_whitelist.append(v)
+        self.allowed_header_names = set(self.default_headers_whitelist)
+
+    def _redact_header(self, key, value):
+        print(value)
+        lower_case_allowed_header_names = [
+            header.lower() for header in self.allowed_header_names
+        ]
+        return value if key.lower() in lower_case_allowed_header_names else self.REDACTED_PLACEHOLDER
```

Comment:
The parent class should already have REDACTED_PLACEHOLDER and can be accessed as HttpLoggingPolicy.REDACTED_PLACEHOLDER here.